website/content/contribute/code/flow/index.md

```diff
@@ -83,41 +83,17 @@ USE_EXISTING_CONTROLLER=true NO_SIMULATED_PROGRESS=true USE_EXISTING_CLUSTER=tru
   should be used. E.g. in integration tests running against an existing setup a controller is already existing
   in the Kubernetes cluster and should not be started by the test as two controllers competing may influence the test.
 
-Setup default local kind cluster with a local image registry:
-```bash
-make kind
-```
-
-By default the docker setup points to the official docker.io registry. When developing
-locally and building/pushing docker containers the docker host and namespace should be changed. When 
-using a local kind setup with a local registry the environment can be pointed to it by setting the following 
-environment variables. (Please note that you also have to add an entry to your /etc/hosts file):
+To support local image registry host resolution append the following to /etc/hosts:
 
-```bash
-export DOCKER_HOSTNAME=kind-registry:5000
-export DOCKER_NAMESPACE=m4d-system
-export HELM_EXPERIMENTAL_OCI=1
 ```
-
-There are make commands for building and pushing docker images separately or in one go:
-```bash
-make docker-build  # Only build docker images
-make docker-push   # Only push docker images to registry defined with env $DOCKER_HOSTNAME
-
-make docker        # Build and push images to the registry defined with env $DOCKER_HOSTNAME
+127.0.0.1       kind-registry
 ```
 
-Deploy on the cluster. This will install the CRDs, dependencies such as the certificate manager and the controller
-itself. The default will pull the images from docker.io. If a local development setup is used please make sure
-that $DOCKER_HOSTNAME is set to the registry that should be used. 
-
-```bash
-make deploy
-```
+With the following you will then setup a kind cluster with the local registry
```

Comment:
Grammar is a bit funky here.  I would suggest something like ... The following command (1) sets up a kind cluster that uses a local registry containing the required images, and then runs a suite of integration tests.  These test check .... _provide information about what is tested_

Reply:
ok based on your suggestion I plan:

The following command sets up a kind cluster that uses a local registry (exposed at kind-registry:5000) containing the required images, and then runs a suite of integration tests. These test check that the m4d core works against  a set of mocked services.